src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,50 @@
+/* TODO: This is just an example file to illustrate API routing and
+documentation. Can be deleted when the first real route is added. */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
```

Comment:
Nitpick: I think it would be better if you grouped the imports (require) at the top, and put this line after the two imports!